R/RtGam.R

```diff
@@ -37,6 +37,10 @@
 #'   [mgcv::bam()] converges more quickly but introduces some additional
 #'   numerical error. Note that the `bam` backend uses the `discrete = TRUE`
 #'   option for an additional speedup. See [mgcv::bam()] for more information.
+#' @param warn_for_diagnostic_failure Should warnings be issued for
+#'   automatically identified diagnostic issues? Defaults to true. A list of
```

Comment:
```suggestion
#'   automatically identified diagnostic issues? Defaults to TRUE. A list of
```

I am not sure that this is good practice but is what I usually do.